### DISCUSSION TITLE
Azure PostgreSQL connection with Azure AD

### DISCUSSION THREAD

**Discussion by danielweil** [prisma/prisma#9953] [Q&A]
Is there any way to use Prisma with Azure PostgreSQL with AD connection? It needs to use a access token as credential for the connection and I was wondering how to proceed with this kind of login, due to the fact the connection string is set as a static config in the schema file.

Any way to manage this?

Faithfully,

**Comment by ruheni** [comment]
Hey @danielweil 👋🏽

Alex from the Prisma team here.

This is not possible at the moment. A workaround would be using the password and username you used to set up the DB server – if this isn't in the organization policy. The only thing you would need to do is add your computer's IP address on the resource on Azure.

You can go ahead and create a [feature request](https://github.com/prisma/prisma/issues) for Postgres and MySQL. One can authenticate with Azure AD when working with SQL Server: https://www.prisma.io/docs/concepts/database-connectors/sql-server#using-integrated-security-windows-only.

![image](https://user-images.githubusercontent.com/33921841/139264354-0e792eaa-6a53-48d3-bad4-507c9267706d.png)

**Comment by timosg** [comment]
I think something similar was discussed here: #4147

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
